README.md

```diff
@@ -772,6 +772,8 @@ type Rules interface {
 	GetStorageValueWriteUnits() uint64 // per chunk
 
 	FetchCustom(string) (any, bool)
+
+	GetMaxActionsPerTx() uint8	
```

Comment:
I think this should be right under `GetValidityWindow`. I purposely tried to leave `FetchCustom` as the last function in `Rules`.